sechubingest.py

```diff
@@ -213,15 +222,15 @@ def _transform_finding(self, vuln):
                 'Product': vuln.get('plugin').get('cvss_base_score',
                                 sevmap[vuln.get('severity_default_id', 0)]),
                 'Normalized': int(vuln.get('plugin').get('cvss_base_score',
-                                sevmap[vuln.get('severity_default_id', 0)]) * 4),
```

Comment:
Setting the top possible vuln rating to 40 for the normalized score is based on feedback from Amazon.  It was originally as you had recommended

From Amazon:
> Findings that are associated with issues that could result in future compromises (e.g., vulnerabilities, configuration weaknesses, exposed passwords). This generally aligns to the Software and Configuration Checks namespace under a finding's type. Suggested score: 1–39. These findings should receive a Low severity label.

Reply:
I've updated this per your feedback. I updated the tests as well.